demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py

```diff
@@ -204,24 +205,92 @@ def test_duplicate_requests_check_duplicates_dont_exist(self):
         }
         duplicate_requests_check(names_dict)
 
+    test_files_path = os.path.join(git_path(), 'demisto_sdk', 'commands', 'postman_codegen', 'tests', 'test_files')
+    with open(os.path.join(test_files_path, 'shared_args_path.json')) as shared_args_path_file:
+        shared_args_path_items = json.load(shared_args_path_file)
+
+    more_then_one_paths = defaultdict(int, {'name': 2, 'description': 3, 'url': 3, 'method': 3})
+    one_for_each_paths = defaultdict(int, {'name': 1, 'id': 1, 'uid': 1})
+    complicated_chars_paths = defaultdict(int, {'name': 1, 'required': 5})
+    same_path_at_the_end = ({'collection.item.settings.name': 'item_settings_name',
```

Comment:
Please change these variables names to also describe what's the object and not only the test case it's related to.

Reply:
Done